Assignment_0/210456_Ishan.md

```diff
@@ -0,0 +1,74 @@
+# HackIT
+
+## UNIX file system
+
+-UNIX file system provides with a way to reach any file on one's PC using Command line interface, it is divided into directories and uses a tree interface to simplify things further
+
+-some useful commands to navigate and interact are:
+
+    cd-command to enter root directory
+        cd.. :to go back to previous directory
+        cd/ : to go to home directory
+    ls-lists components of folder alone
+        ls -l: long listing of files in directory
+        ls -a: causes hidden files to be listed
+    man 'command': manual to 'command'
+        man ls: gives manual to ls command
+    rm: deletes specific file
+        rm xyz: deletes file named xyz
+        rm xyz abc: deletes file named xyz and abc
+        rm xyz*: deletes files with xyz prefix
+    cp: to copy files and and even whole directories
+    grep: to find a given line
+        just like the one we used in Q3
+    sudo:gives privileges of another user (often the superuser)
+        sudo apt-get xyz
+    su: is used to give commands as another user
+        su user -c command
+    pwd: locating the directory currently placed in
+    mkdir: creates new folder
+
+
+
+---
+
+## vim
+
+-vim is a very versatile and a lightweight program, althhoughh it might look hard to use on the surface but with enough practice and getting used to it can increase productivity by a lot
```

Comment:
mild spelling errors. These don't matter, but can be and should be avoided while writing a writeup/documentation.